lib/cisco_node_utils/interface_hsrp_group.rb

```diff
@@ -393,6 +474,10 @@ def priority_level_set(pri, lower, upper)
       config_set('interface_hsrp_group', 'priority_level', @set_args)
     end
 
+    # This CLI can take forms like:
```

Comment:
I did slightly different but used regex nevertheless. It is working now, please let me know if this is fine.